db/migrate/20151111030421_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.text :body
+      t.references :post, index: true, foreign_key: true
```

Comment:
Cool, thanks. Migrations are and area I'm working on.

Found a cool GUI-based app the imports/exports Rails migrations, but probably overkill for what we are doing - https://www.malcolmhardie.com/sqleditor/